### DISCUSSION TITLE
Thirdweb Vulnerability

### DISCUSSION THREAD

**Discussion by sbvegan** [ethereum-optimism/developers#111] [Announcements]
Hello all! Our friends at [Thirdweb](https://thirdweb.com/) informed our team about a security vulnerability in a commonly used open source library, impacting contracts across the web3 ecosystem.

Based on the information we have received, there is no evidence that any Superchain protocol contracts are impacted.

We have been in close contact with the team on mitigation plans, which you can read more about on their [blog](https://blog.thirdweb.com/security-vulnerability/).

**Comment by sbvegan** [comment]
Details on the vulnerability can be found on the following Open Zeppelin [blog post](https://blog.openzeppelin.com/arbitrary-address-spoofing-vulnerability-erc2771context-multicall-public-disclosure)

**Comment by opfocus** [comment]
Stay tuned
